Tests/ML/histopathology/models/test_encoders.py

```diff
@@ -0,0 +1,76 @@
+import os
+from pathlib import Path
+from typing import Callable
+
+import pytest
+from torch import Tensor, float32, nn, rand
+from torchvision.models import resnet18
+
+from health_azure.utils import CheckpointDownloader, get_workspace
+from InnerEye.Common import fixed_paths
+from InnerEye.ML.Histopathology.models.encoders import (TileEncoder, HistoSSLEncoder, ImageNetEncoder,
+                                                           ImageNetSimCLREncoder, InnerEyeSSLEncoder)
+
+
+def get_supervised_imagenet_encoder() -> TileEncoder:
+    return ImageNetEncoder(feature_extraction_model=resnet18, tile_size=224)
+
+
+def get_simclr_imagenet_encoder() -> TileEncoder:
+    return ImageNetSimCLREncoder(tile_size=224)
+
+
+def get_simclr_crck_encoder(tmp_path: Path) -> TileEncoder:
+    aml_workspace = get_workspace()
+    download_dir = tmp_path / "downlads"
+    download_dir.mkdir(exist_ok=True)
+    downloader = CheckpointDownloader(aml_workspace=aml_workspace,
+                                      run_id="vsalva_ssl_crck:vsalva_ssl_crck_1630691119_af10db8a",
```

Comment:
just a note to say that this test will not work in a different workspace - we could skip maybe based on the workspace?

Reply:
Removed the test from this PR